codalab/worker_manager/worker_manager.py

```diff
@@ -124,19 +125,24 @@ def run_loop(self):
             try:
                 self.run_one_iteration()
             except (
-                urllib.error.URLError,
                 http.client.HTTPException,
                 socket.error,
                 NotFoundError,
-                JsonApiException,
             ):
                 # Sometimes, network errors occur when running the WorkerManager . These are often
                 # transient exceptions, and retrying the command would lead to success---as a result,
                 # we ignore these network-based exceptions (rather than fatally exiting from the
                 # WorkerManager )
                 traceback.print_exc()
+            except (urllib.error.URLError, JsonApiException, ssl.SSLError) as e:
+                if "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed" in str(e):
```

Comment:
Is there a cleaner way to check if the user entered incorrect login info? Is it possible to check the type of exception instead?

Reply:
The reason that I do this is that in case that if there are other places throwing these types of errors, we don't want to handle in the same way, as the causes could be different.

Reply:
I'm not sure what you mean by same way. Can you just check the exception type instead or is that not possible?

Reply:
It is possible but I mean, if there are somewhere else throw the same type of error due to some other reason, do we still want to show this message? It might be not accurate.